internal/commands/logs/list_inputs_test.go

```diff
@@ -27,5 +29,24 @@ func TestLogTypes(t *testing.T) {
 
 			assert.Equal(t, tc.logTypes, i.logTypes())
 		})
+
+		t.Run("should fail if log type "+tc.logType+" is not formatted before passed to client", func(t *testing.T) {
```

Comment:
I think this test would be better suited for `list_test.go`, since its running the `cmd.Handler` function (it can be based on just one of these test cases)

---

also [nit] while string concatenation like this is fairly performant in go, considering we're dealing with test descriptions, i'd just use `fmt.Sprintf(...)` here as it reads a bit easier